### PR TITLE
Refactor/add child data get

### PR DESCRIPTION
**Pull request by bramirez96**
# Description

Adds update to Add Child form to allow selection box values to be populated from the database. This fix ensures data integrity and makes maintenance easier, with no noticeable slowdown of the application.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [ ] No
- [x] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/api/index.js [hunk 1]
```diff
@@ -66,6 +66,22 @@ const getProfileData = authState => {
   }
 };
 
+const getChildFormValues = async authState => {
```
Comment (Schrese) [inline src/api/index.js:74]: Is this for the child avatars?

File: src/components/pages/AddChild/RenderAddChild.js [hunk 2]
```diff
@@ -103,54 +98,38 @@ const RenderAddChild = props => {
...
           </Form.Item>
           <Form.Item {...dyslexia}>
             <Form.Item
               name="IsDyslexic"
-              onChange={handleChange}
               label="Dyslexia"
               valuePropName="checked"
             >
-              <Switch
-                // checkedChildren="On"
```
Comment (SandraCoburn) [inline src/components/pages/AddChild/RenderAddChild.js:149]: At some point we want to put these two items back.

**Review by SandraCoburn — DISMISSED** [review]
Nicely done!

**Review by Nick-Ohman — DISMISSED** [review]
Looks great!

File: src/api/index.js (hunk 1 above)
Comment (bramirez96) [reply src/api/index.js:74]: Loads potential form data from the database!

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Review by JLC6290 — APPROVED** [review]
Looks great!

**Review by Schrese — APPROVED** [review]
Looks Great!

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Pull request merged**